pkg/opssight/ctl_opssight.go

```diff
@@ -506,7 +502,7 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 			ctl.Spec.Perceiver.Port = ctl.PerceiverPort
 		case "configmapname":
 			ctl.Spec.ConfigMapName = ctl.ConfigMapName
-		case "secretname":
+		case "secret-name":
```

Comment:
Why is this secret-name but configmap is configmapname?  Need consistency in option names.

Reply:
fixed